01-intro-to-R.Rmd

```diff
@@ -316,9 +315,11 @@ from. The other 4 are:
   `1+4i`) and that's all we're going to say about them
 * `"raw"` that we won't discuss further
 
+A particular and very useful data type is `"factor"` which encodes categorical variables (e.g. Habitat types, Gender etc.) 
```

Comment:
Agreed, you are obviously technically right, although from a pedagogic point of view, I find this is quite confusing for the students, as you can define variables within data frames as factors, exactly as you can define them as numeric, integer or character. but I get your point!